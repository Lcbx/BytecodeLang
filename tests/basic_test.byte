-8*5
13. + 1020.
3242 # hi
'great'
'\'I love RocknRoll\''
8*3/2-5*2
5<4
17==17
5<4!=34
4<6 and 45==45 and 23==32
23==32 and 45==45 and 4<6
1024

