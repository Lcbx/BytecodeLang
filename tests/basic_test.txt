hi hello2
hi 13. +1020
hi 3242 # hi
hi "great"
hi "\"I love RocknRoll\""
hi -8*5.
hi 8*3/2-5*2
hi 5<4
hi 17==17
hi 5<4!=34
hi 4<6 and 45==45 and 23==32
hi 23==32 and 45==45 and 4<6
hi 1024

