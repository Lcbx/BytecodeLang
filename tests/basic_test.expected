compile_Out:
Error line 1, column 4 : hello2 not declared yet
1 errors
__SECTION__
compile_Err:

__SECTION__
simulation_Out:

__SECTION__
simulation_Err:
Traceback (most recent call last):
  File "C:\Users\lucco\Desktop\tests\Lang\compiler\vm_simulator.py", line 13, in <module>
    with open(args.input,'rb') as file:
FileNotFoundError: [Errno 2] No such file or directory: 'tests\\basic_test.hex'